runtime/compiler/net/ClientStream.cpp

```diff
@@ -75,13 +76,20 @@ void ClientStream::static_init(TR::PersistentInfo *info)
       exit(1);
       }
 
-   auto &sslKeys = info->getJITaaSSslKeys();
-   auto &sslCerts = info->getJITaaSSslCerts();
-   auto &sslRootCerts = info->getJITaaSSslRootCerts();
-
    // TODO: we could also set up keys and certs here, if it's necessary to use TLS client keys 
    // it's not needed for a basic setup.
-   TR_ASSERT_FATAL(sslKeys.size() == 0 && sslCerts.size() == 0, "client keypairs are not yet supported, use a root cert chain instead");
+   TR::CompilationInfo *compInfo = TR::CompilationInfo::get();
+   auto &sslRootCerts = compInfo->getJITServerSslRootCerts();
+
+#if defined(DEBUG)
+   const PersistentVector<std::string> *sslKeys = compInfo->getJITServerSslKeys();
+   const PersistentVector<std::string> *sslCerts = compInfo->getJITServerSslCerts();
+
+   if (sslKeys && sslCerts)
+      {
+      TR_ASSERT_FATAL(sslCerts->size() == 0 && sslCerts->size() == 0, "client keypairs are not yet supported, use a root cert chain instead");
```

Comment:
sslCerts->size() appears twice
So, we setup these vectors, but in reality we don't support any of them.